packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -52,7 +52,9 @@ class RequestQueue {
    */
   async initQueue() {
     const idbQueue = await this._idbQDb.get(this._queueName);
-    this._queue.concat(idbQueue);
+    if (idbQueue) {
```

Comment:
Yeah, as I mentioned in the PR description, I needed to include this to make the tests pass. What's strange is why it's still showing up in the diff since his change was merged.

I can take a look before the final merge.